src/Utils.ts

```diff
@@ -102,10 +102,19 @@ export default class Utils {
     if (vorpal._command &&
       vorpal._command.args &&
       vorpal._command.args.options &&
-      vorpal._command.args.options.output === 'json') {
+      vorpal._command.args.options.output === 'json' &&
```

Comment:
To avoid duplicating the conditions, we could simplify the block to:

```ts
if (vorpal._command &&
  vorpal._command.args &&
  vorpal._command.args.options &&
  vorpal._command.args.options.output === 'json') {
  if (vorpal._command.args.options.pretty) {
    return JSON.stringify(logStatement, null, 2);
  }
  else {
    return JSON.stringify(logStatement);
  }
}
```